# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/xul/nsXULTooltipListener.cpp
# Commit: 2adbab502a1e
# Full Hash: 2adbab502a1e690fccd61fb604be820e60d61723
# Author: Kristen Wright <kwright@mozilla.com>
# Date: 2020-09-09 09:39:57
# Regressor Bug: 1642727
# File Overlap Count: 1
# Description:
#   Bug 1642727 - Remove the VarCache machinery r=njn
#   
#   This commit removes Varcache entirely, putting an end to our quest to get rid of VarCache.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D79538
# ==============================================================================

diff -r 1aeb7707569f -r 2adbab502a1e layout/xul/nsXULTooltipListener.cpp
--- a/layout/xul/nsXULTooltipListener.cpp	Tue Sep 08 15:05:06 2020 +0000
+++ b/layout/xul/nsXULTooltipListener.cpp	Tue Sep 08 17:12:01 2020 +0000
@@ -53,7 +53,7 @@
       mLastTreeRow(-1)
 #endif
 {
-  // FIXME(emilio): This can be faster, this should use BoolVarCache.
+  // FIXME(emilio): This can be faster, this should use static prefs.
   //
   // register the callback so we get notified of updates
   Preferences::RegisterCallback(ToolbarTipsPrefChanged,